src/jobs/gradle_owasp_dependency_check.yml

```diff
@@ -29,6 +29,9 @@ parameters:
     type: string
     default: dps_alerts_security
     description: Slack channel to use for notifications.
+  jdk_tag:
+    default: "17.0"
```

Comment:
just wondering if there is any harm bumping this to 19 anyway? it's not going to affect the outcome of the scan result is it?